### DISCUSSION TITLE
Password on Opening

### DISCUSSION THREAD

**Discussion by yerch1982** [jendrikseipp/rednotebook#585] [Ideas]
Some of the things on the journal are very personal so I was thinking on a password on entering the journal :)

**Comment by z0diaq** [comment]
Good idea for new feature IMO - but keep in mind RNB keeps data in plaintext so it's still easily readable outside of app...

On other hand could not find any issue related to encrypting source data, wouldn't that make also nice new feature ?

**Reply by jendrikseipp** [reply]
I think it's better to use external tools for this purpose, such as encrypting your filesystem.

**Comment by laraconda** [comment]
If you are using linux, you could use GPG to encrypt your entry files before opening red notebook, once you are done writing, the files go back to being encrypted.
This process can be automated. Ideally, you make a wrapper around red notebook that:

1. decrypts
2. runs red notebook
3. on red notebook closing, encrypts the files again.

The first two steps are not a problem. The third would be my only concern but maybe it is easy enough to do.

**Comment by m3thm4th** [comment]
This is only part of what's required in order to make this app private. I opened #761 to discuss such issues.
